cachito/workers/pkg_managers/pip.py

```diff
@@ -1514,3 +1514,64 @@ def upload_raw_package(repo_name, artifact_path, dest_dir, filename, is_request_
     to_nexus_hoster = not is_request_repository
     log.debug("Uploading %r as a raw package to the %r Nexus repository", artifact_path, repo_name)
     nexus.upload_raw_component(repo_name, dest_dir, components, to_nexus_hoster)
+
+
+def get_pypi_hosted_repo_name(request_id):
+    """
+    Get the name of the PyPI hosted repository for the request.
```

Comment:
Nitpicky suggestion (optional): `Get the name of the Nexus PyPI hosted repository for the request.`